### PR TITLE
Update pcoords.rst

### PR DESCRIPTION
**Pull request by pswaldia**
In the documentation of Parallel Coordinates, one of the possible value for normalize parameter is mentioned as 'minabs' , which is not being implemented in yellowbrick.  Current supported options are 'minmax', 'maxabs', 'standard', 'l1'  and 'l2' according to _pcoords.py_ . This PR corrects the documentation in refrence to issue #797. The reason for non inclusion of 'minabs' is the non availability of _MinAbsScaler_ in _sklearn.preprocessing_ .

### CODE REVIEW

**Comment by rebeccabilbro** [comment]
Thanks so much for the quick response here and on #797 @pswaldia!  @haleemason's got us covered & the fix is thankfully merged in now, so going to close this PR. If you ever get a hankering to implement `minabs`; let us know!!
